packages/webcomponents/src/components/form/form-control-monetary.tsx

```diff
@@ -11,40 +11,39 @@ import IMask, { InputMask } from 'imask';
 import { CURRENCY_MASK } from '../../utils/form-input-masks';
 
 @Component({
-  tag: 'form-control-monetary',
```

Comment:
@colganmeanor one thing that I would suggest is to follow the component organization order suggested on Stencil docs. https://stenciljs.com/docs/style-guide

It makes easier to follow lifecycle methods, watchers and so on.

Reply:
Good call! Just pushed up an update to match the guide you referenced 👍🏻